Website/src/rendering/src/pages/shop/checkout/cart.tsx

```diff
@@ -11,19 +16,32 @@ interface CartProps {
 }
 
 const Cart = (props: CartProps): JSX.Element => {
+  const { order } = useOcCurrentOrder();
+  const breadCrumbDefinitions: Breadcrumb[] = [
+    { urlPath: '/shop', displayName: 'Shop' },
+    { urlPath: '/cart', displayName: 'Cart' },
+  ];
   return (
     <ShopLayout>
       {console.log({ props })}
       <Head>
         <title>PLAY! SHOP - Cart</title>
       </Head>
 
-      <p>Cart Page</p>
+      <Breadcrumb
+        rendering={{ componentName: '' }}
+        params={{}}
+        fields={{ items: breadCrumbDefinitions }}
+      />
+      <h1>Your Shopping Cart</h1>
```

Comment:
@NellyKostadinova, I know it is work in progress. Will this page be modified to use `CartDetails` instead of individual components?